ui/admin/app/templates/scopes/scope/credential-stores/index.hbs

```diff
@@ -6,8 +6,11 @@
   <page.header>
     <h2>
       {{t 'resources.credential-store.title_plural'}}
+      <DocLink @doc='credential-stores' @iconSize='large' />
     </h2>
-    {{t 'resources.credential-store.description'}}
+    {{#if @model}}
```

Comment:
`@model` check isn't needed when displaying plain text.

Reply:
That is right! good catch thank you!